### PR TITLE
Updated Repo and API READMEs

### PR DESCRIPTION
**Pull request by dxslly**
- Updated the README to reflect the beta production ready status.
 - Fixed example gradle dependency in API README.

### CODE REVIEW

**Comment by dxslly** [comment]
Requesting @nerdyverde and @StanKocken to review.

File: README.md [hunk 1]
```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
...
-## OpenYOLO is *not* production ready
+## Project Status
 
-OpenYOLO is currently in the *experimental* stage; both the specification and
-reference implementation are *unstable*. We do not recommend that it be used
-in production Android apps. At such a time that OpenYOLO is considered stable
-and ready for production use, an official announcement shall be made and both
-the specification and reference implementation will be promoted to v1.0.0.
+OpenYOLO has reached beta production ready stage and can be integrated into production applications.
+There are currently several production implementation of credential providers including:
```
Comment (nerdyverde) [inline README.md]: This phrase might be more readable as

```The following credential providers include production implementations of OpenYOLO:```

At the very least, we seem to be missing an 's' at the end of "implementation".

File: README.md [hunk 2]
```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
...
-The OpenYOLO for Android specification can be found 
-[here](https://spec.openyolo.org/openyolo-android-spec.html). This is
-currently under review by the 
-[Account Chooser and OpenYOLO Working Group](http://openid.net/wg/ac/),
-with the intention for it to become an implementor's draft by Q3 2017,
-and reach full standardization by Q4 2017.
+The OpenYOLO for Android specification can be found [here](http://openid.net/specs/openyolo-android-03.html).
+This is currently under review by the [Account Chooser and OpenYOLO Working Group](http://openid.net/wg/ac/),
+and has reached implementor's draft with the intention for it to reach full standardization in the
+near future.
```
Comment (nerdyverde) [inline README.md]: What about the following to keep the terms consistent with OpenID?

```This is currently under review as an implementor's draft by the [Account Chooser and OpenYOLO Working Group](http://openid.net/wg/ac/), with the goal for it to reach final specification status in the near future.```

File: README.md [hunk 3]
```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
 This repository contains the reference implementation of the OpenYOLO protocol,
 and the standard API and Service Provider Interface (SPI).
 
-OpenYOLO is inspired by, and ultimately intended to replace, Google's
-[Smart Lock for Passwords][yolo] API, which is
-internally known as YOLO (You Only Login Once).
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/445775/30930082-60d21ace-a375-11e7-90cf-cc75fe5747fd.gif"/>
```
Comment (iainmcgin) [inline README.md]: You should be able to Markdown-ify this image:

```
![Animation demonstrating auto sign-in](https://user-images.githubusercontent.com/445775/30930082-60d21ace-a375-11e7-90cf-cc75fe5747fd.gif)
```

File: README.md [hunk 4]
```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
...
-OpenYOLO is currently in the *experimental* stage; both the specification and
-reference implementation are *unstable*. We do not recommend that it be used
-in production Android apps. At such a time that OpenYOLO is considered stable
-and ready for production use, an official announcement shall be made and both
-the specification and reference implementation will be promoted to v1.0.0.
+OpenYOLO has reached beta production ready stage and can be integrated into production applications.
+There are currently several production implementation of credential providers including: 
 
-## OpenYOLO Specification
+| <img src="https://user-images.githubusercontent.com/445775/31100963-59c51db2-a780-11e7-87f4-e7ab2750b92a.png" width="200" height="200" /> | <img src="https://user-images.githubusercontent.com/445775/31101083-0cc4fd92-a781-11e7-95fe-9f9e9fadae70.png" width="200" height="200" /> | <img src="https://user-images.githubusercontent.com/445775/31101050-eeb9966e-a780-11e7-96ae-614f765f3f12.png" width="200" height="200" /> |
```
Comment (iainmcgin) [inline README.md]: Markdown-ify, plx

File: README.md [hunk 5]
```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
...
-with the intention for it to become an implementor's draft by Q3 2017,
-and reach full standardization by Q4 2017.
+The OpenYOLO for Android specification can be found [here](http://openid.net/specs/openyolo-android-03.html).
+This is currently under review by the [Account Chooser and OpenYOLO Working Group](http://openid.net/wg/ac/),
+and has reached implementor's draft with the intention for it to reach full standardization in the
+near future.
+
+## Getting Started as a Client
+
+[ ![Download](https://api.bintray.com/packages/openid/net.openid/openyolo-api/images/download.svg?version=0.2.0) ](https://bintray.com/openid/net.openid/openyolo-api/0.2.0/link)
```
Comment (iainmcgin) [inline README.md]: There's a variant of this that always points to latest:

```
[![Download](https://api.bintray.com/packages/openid/net.openid/openyolo-api/images/download.svg) ](https://bintray.com/openid/net.openid/openyolo-api/_latestVersion)
```

You're actually already using this further down, so better to be consistent than pin it to a particular version.

File: README.md [hunk 6]
```diff
@@ -11,25 +11,50 @@ any installed credential provider on the device.
...
+
+The open-source client library for the OpenYOLO for Android protocol lives in the ```api```
```
Comment (iainmcgin) [inline README.md]: You can omit "open-source" here, as it's redundant info.

File: README.md (hunk 3 above)
Comment (dxslly) [reply README.md]: I had tried a few times, but was not able to also center the image.

File: README.md (hunk 1 above)
Comment (dxslly) [reply README.md]: Thanks, done.

File: README.md (hunk 4 above)
Comment (dxslly) [reply README.md]: I also tried using markdown here, but was unable to specify image size constraints. If we'd like to keep this file strictly markdown and I can try and work around this (By resizing the images) and reformat the page to remove html. IMO it is worth using the html tags for the extra styling control at the cost of reducing human readability of the raw file and possible lack of support if rendered outside of github.

File: README.md (hunk 2 above)
Comment (dxslly) [reply README.md]: Sgtm, done.

File: README.md (hunk 5 above)
Comment (dxslly) [reply README.md]: Ops, good catch, fixed.

File: README.md (hunk 6 above)
Comment (dxslly) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (iainmcgin) [reply README.md]: You worry too much about formatting :rofl:

**Comment by dxslly** [comment]
@nerdyverde PTAL

**Comment by iainmcgin** [comment]
I'd go ahead and submit this as you have LGTM from Stan, we can handle any comments from Michael in follow-up.

**Comment by nerdyverde** [comment]
No need for follow-up. LGTM :)

**Pull request merged**